CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift

```diff
@@ -22,6 +27,15 @@ final class CaptureSuggestionsViewContainer: UIView {
 
     override func awakeFromNib() {
         super.awakeFromNib()
+
+        isAccessibilityElement = false
```

Comment:
@vizaknaidavid would you mind accessibility-related code in a separate method to `configure Accessibility()` ?